src/draw/lv_draw_buf.h

```diff
@@ -152,6 +159,70 @@ void lv_draw_buf_clear(void * buf, uint32_t w, uint32_t h, lv_color_format_t col
 void lv_draw_buf_copy(void * dest_buf, uint32_t dest_w, uint32_t dest_h, const lv_area_t * dest_area_to_copy,
                       void * src_buf,  uint32_t src_w, uint32_t src_h, const lv_area_t * src_area_to_copy,
                       lv_color_format_t color_format);
+
+/**
+ * Note: Eventually, lv_draw_buf_malloc/free will be kept as static.
+ *       For now, we use `create` to distinguish with malloc.
+ *
+ * Create an draw buf by allocating struct for `lv_draw_buf_t` and allocating a buffer for it
+ * that meets specified requirements.
+ *
+ * @param w         the buffer width in pixels
+ * @param h
+ * @param cf        the color format for image
+ * @param buf_align the start address alignment for the buffer
```

Comment:
I think it  `LV_DRAW_BUF_ALIGN` should be used internally instead of having a dedicated parameter. 

Or are there any advantages of allocating a not align or differently aligned buffer?

Reply:
Two draw unit that may have different requirements on stride like CPU91) and GPU(64).
Include `buf_align` provides us best flexibility, and we can also provide API using default `buf_align`.

Reply:
That also reminds me default buf_align should be bound with every draw unit, instead of a global config.